src/component.ts

```diff
@@ -31,10 +32,11 @@ export class StakingContractsComponent implements Component {
     RirisuStakingContractAdapter,
     RoboStakingContractAdapter,
     SkyFarmStakingContractAdapter,
+    UwULendStakingContractAdapter,
     DogsUnchainedStakingContractAdapter,
     // NFTWEscrowStakingContractAdapter,
     PerionCreditsStakingContractAdapter,
     // DigitzStakingContractAdapter,
   ];
-  constructor() {}
+  constructor() { }
```

Comment:
Please revert this code formatting change. You can run `npm run prettier:fix` to make sure.

Reply:
Sure, think.